pytorch_lightning/accelerators/gpu.py

```diff
@@ -53,6 +57,78 @@ def set_nvidia_flags(local_rank: int) -> None:
         devices = os.getenv("CUDA_VISIBLE_DEVICES", all_gpu_ids)
         _log.info(f"LOCAL_RANK: {local_rank} - CUDA_VISIBLE_DEVICES: [{devices}]")
 
+    def get_device_stats(self, device: Union[str, torch.device]) -> Dict[str, Any]:
+        """Gets stats for the given GPU device.
+
+        Returns:
+        A dictionary mapping the metrics to their values.
```

Comment:
```suggestion
            A dictionary mapping the metrics to their values.
```
needs one tab indentation